Create CCC_TAC_Minutes-2022-06-02.md (#114)
confidential-computing/governance: TAC/Meetings/2022/2022-06-02/CCC_TAC_Minutes-2022-06-02.md

#### Voting members
* Dave Thaler (Microsoft)
* Lily Sturmann (Red Hat)
* Dan Middleton (Intel)

#### Other Attendees
* Eric Voit (Cisco)
* Nagaraju (Intel)
* Megan Collier (Investment Group)
* Matthieu Legre (Cysec)
* Nick Vidal (Profian)
* Shalom Shefa (Cisco)
* Mark Novak (JP Morgan Chase)
* Jethro Beekman (Fortanix)
* Ravi Sahita (Rivos)
* Xinxin Fan (Iotex.io)
* Henk Birkholz (Fraunhofer)
* Thomas Hardjono (MIT)
* Penglin Yang (China Mobile)
* Hannes Tschofenig (Arm)
* Kurt Taylor (Linux Foundation)
* Helen Lau (Linux Foundation)

### Meeting minute approvals
#### Approved by voting members

**Proposed:** That the minutes of the [April 21, 2022](../2022-04-21/TAC_Minutes-2022-04-21.pdf) and [May 19, 2022](../2022-05-19/TAC_Minutes-2022-05-19.pdf) meeting of the Technical Advisory Council meeting of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.

 * Quorum was not reached and the minutes were not approved. 
 
#### Pending approval
 * [April 21, 2022](../2022-04-21/TAC_Minutes-2022-04-21.pdf)
 * [May 19, 2022](../2022-05-19/TAC_Minutes-2022-05-19.pdf)

### TAC Tech Talks

>  *  Multi-TEE Systems: PCI-SIG WG Update

Author: ```suggestion
 *  Multi-TEE Systems: [PCI-SIG WG Update](https://github.com/confidential-computing/governance/blob/main/TAC/Meetings/2022/2022-06-02/Towards%20Attesting%20Multi-TEE%20Devices.pptx)
```

>  *  Presented by - Mark Novak (MN)

Reviewer: Add link to slides

  *  [Management Component Transport Protocol (MCTP)](https://en.wikipedia.org/wiki/Management_Component_Transport_Protocol)
 *  DT asked for possible collaboration with the IETF

### Open Action Items
1. [Stephen, Zongmin] Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
 * Please report back that project maintainers have been made aware of the training, and have been asked to take it.
2. [Mentors] Mentors to reach out to the project for getting the maintainers involved in Common Test Infrastructure
3. [Thomas F/Kurt] Update GitHub with Veraison charter
4. [Mike Bursell] Wikipedia update needed - Trusted Computing wiki page
5. [Kurt] to help Dave find discussion on governance subgroup of people in order to get an update on what they have completed (Done)
6. [Mark N, Thomas F, Naveen C, Steve VL, Eric N] governance subgroup - collaborating on the governance topic, SIG formation and status update requested
7. [Kurt] to have LFCS proceed with draft conversion and formatting of white paper PDF (Done)

> 8. [Mentors] Code scanning via LFX Security - add projects to security for scanning with GitHub bot (Set up the bot by following these [instructions](https://community.lfx.dev/t/how-to-install-and-configure-bots-to-secure-your-projects/181)

Reviewer: ```suggestion
8. [Mentors] Code scanning via LFX Security - add projects to security for scanning with GitHub bot (Set up the bot by following these [instructions](https://community.lfx.dev/t/how-to-install-and-configure-bots-to-secure-your-projects/181))
```
However maybe this line should be deleted as it duplicates line 110-111

    
 * DT - discussed [#90](https://github.com/confidential-computing/governance/pull/90) another that DM needs to comment on
 * DT - discussed the charter Project Progression Policy stages as background for Issue [#104](https://github.com/confidential-computing/governance/issues/104), that everything in [Graduated](https://github.com/confidential-computing/governance/blob/main/project-progression-policy.md#graduation-stage) and [Emeritus](https://github.com/confidential-computing/governance/blob/main/project-progression-policy.md#emeritus-stage) has never been voted on, need to revisit and update or accept as is, low priority

### Any other business
* DT reviewed upcoming presentations
  * PCI SIG WG was presented this meeting
  * Enclave device blueprints for Confidential Computing at the edge - June 30
  * Multi-TEE systems (e.g., CPU+GPU+NIC TEEs)
	 * Trust Domains - Mike Bursell possibly July 14th
* No presentation for next meeting June 16th

* KT asked about any upcoming face to face meeting, NV said next was Blackhat 
* DT asked if others were attending IETF in Philadelphia 23 - 24, July 2022 (Saturday - Sunday)

>   * DT noted there may be an opportunity to meetup at [IETF Hackathon](https://www.ietf.org/how/runningcode/hackathons/114-hackathon/)

Reviewer: ```suggestion
  * DT noted there may be an opportunity to meet up at [IETF Hackathon](https://www.ietf.org/how/runningcode/hackathons/114-hackathon/)
```
